tools/build/wheel_linux_before_all.sh

```diff
@@ -1,25 +1,27 @@
-#!/bin/bash
+#!/bin/sh
 
 # Configure the libraries needed to build wheel packages on linux.
 # This script is designed to be used by cibuildwheel as CIBW_BEFORE_ALL_LINUX
 
-set -euo pipefail
-set -x
+set -xe
 
-dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+. /etc/os-release
 
-# Install libpq
-# Note that the pgdg doesn't have an aarch64 repository so wheels are build
-# with the libpq packaged with Debian 9, which is 9.6.
-if [[ ! "$AUDITWHEEL_ARCH" = "aarch64" ]]; then
-    source /etc/os-release
-    echo "deb http://apt.postgresql.org/pub/repos/apt ${VERSION_CODENAME}-pgdg main" \
-        > /etc/apt/sources.list.d/pgdg.list
-    # TODO: on 2021-09-30 curl fails with
-    # curl: (60) SSL certificate problem: certificate has expired
-    # Test again later if -k can be removed.
-    curl -sk https://www.postgresql.org/media/keys/ACCC4CF8.asc \
-        | apt-key add -
-fi
-apt-get update
-apt-get -y install libpq-dev
+# Install PostgreSQL development files.
+case "$ID" in
+    alpine)
+        apk add --no-cache postgresql-dev
+        ;;
+    debian)
+        # Note that the pgdg doesn't have an aarch64 repository so wheels are
+        # build with the libpq packaged with Debian 9, which is 9.6.
+        if [ "$AUDITWHEEL_ARCH" != 'aarch64' ]; then
+            echo "deb http://apt.postgresql.org/pub/repos/apt $VERSION_CODENAME-pgdg main" \
+                > /etc/apt/sources.list.d/pgdg.list
+            curl -sf https://www.postgresql.org/media/keys/ACCC4CF8.asc \
+                > /etc/apt/trusted.gpg.d/postgresql.asc
+        fi
+        apt-get update
+        apt-get -y install libpq-dev
+        ;;
+esac
```

Comment:
I think we would like an else case to explode with incommensurable noise here

Reply:
Is this good enough?

```diff
diff --git a/tools/build/wheel_linux_before_all.sh b/tools/build/wheel_linux_before_all.sh
index 11570d6..a7fd911 100755
--- a/tools/build/wheel_linux_before_all.sh
+++ b/tools/build/wheel_linux_before_all.sh
@@ -24,4 +24,8 @@ case "$ID" in
         apt-get update
         apt-get -y install libpq-dev
         ;;
+    *)
+        echo "Unknown Linux distribution: '$ID'" >&2
+        exit 1
+        ;;
 esac
```

Reply:
"Unexpected" is more correct than "Incorrect", for the rest yes, that will do :)